### PR TITLE
Deobfuscated some stuff

### PR DESCRIPTION
**Pull request by darkerbit**

### CODE REVIEW

File: mappings/net/minecraft/block/dispenser/DispenserSoundBehaviour.mapping [hunk 1]
```diff
@@ -0,0 +1,2 @@
+CLASS fs net/minecraft/block/dispenser/DispenserSoundBehaviour
```
Comment (asiekierka) [inline mappings/net/minecraft/block/dispenser/DispenserSoundBehaviour.mapping]: Behavior

File: mappings/net/minecraft/block/dispenser/DispenserSoundBehaviour.mapping [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+	FIELD b eventType Z
```
Comment (asiekierka) [inline mappings/net/minecraft/block/dispenser/DispenserSoundBehaviour.mapping]: Boolean? Type?

**Review by asiekierka — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/block/dispenser/DispenserSoundBehaviour.mapping (hunk 2 above)
Comment (darkerbit) [reply mappings/net/minecraft/block/dispenser/DispenserSoundBehaviour.mapping]: This is used in a ternary operator between event types 1000 and 1001.

File: mappings/net/minecraft/block/dispenser/DispenserSoundBehaviour.mapping (hunk 1 above)
Comment (darkerbit) [reply mappings/net/minecraft/block/dispenser/DispenserSoundBehaviour.mapping]: Fixed

File: mappings/net/minecraft/block/dispenser/DispenserSoundBehaviour.mapping (hunk 2 above)
Comment (darkerbit) [reply mappings/net/minecraft/block/dispenser/DispenserSoundBehaviour.mapping]: Renamed as eventTypeSelector

Comment (Gegy) [reply mappings/net/minecraft/block/dispenser/DispenserSoundBehaviour.mapping]: What does each event type represent?

**Review by modmuss50 — APPROVED** [review]

File: mappings/net/minecraft/block/dispenser/DispenserSoundBehaviour.mapping (hunk 2 above)
Comment (Gegy) [reply mappings/net/minecraft/block/dispenser/DispenserSoundBehaviour.mapping]: I think this name should reflect what the event types are

**Comment by modmuss50** [comment]
Sorry about leaving this PR behind in the update, we forgot in the chaos.

I will redo this PR later today, as updating it (across 2 versions) is a pain.

**Pull request closed without merging**